spec/features/proposal/manage_collection_proposals/cannot_manage_cmr_spec.rb

```diff
@@ -0,0 +1,32 @@
+describe 'Attempting to access the Manage CMR functionalities', js: true do
```

Comment:
I think the test description should say that we are in Draft MMT or proposal mode